### PR TITLE
hey i have made some tweaks on your's project

### PR DESCRIPTION
**Pull request by ghost**
after seeing your  project  i thought that i can contribute on your project by giving your nav bar with  great UI so i hope you will like these tweaks and will accept my PR so i will wait for your response and feel free to give any  suggestions that will be a great opportunity to improve myself..

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -40,15 +38,24 @@
...
 </head>
 
-<body text="#000000" topmargin="0" leftmargin="0" rightmargin="0" bottommargin="0" marginheight="0" marginwidth="0"
-    onload="document.textform.text.focus();">
+<body text="#000000" topmargin="0" leftmargin="0" rightmargin="0" bottommargin="0" marginheight="0" marginwidth="0" onload="document.textform.text.focus();">
+
+
+    <div id="nav">
+        <div id="logo"><span style="font-weight:bold; font-size:25px;color:#ff3333">Edit</span><span>Pad</span></div>
+    </div>
```
Comment (shweshi) [inline src/index.html]: @Ashish9836 This nav is not looking good with the current look of the Editpad. We can get rid of this.

**Comment by shweshi** [comment]
@Ashish9836 hey, thanks for the wonderful tweaks. Here are my thoughts on same.
1. The navbar is not matching with the current aesthetics of the Editpad. We can remove it.
2. The buttons are getting so prominent and snatching the focus from the actual text area. I was thinking that when user is using the app main highlight or focus area should be on textarea, and the navbar button should not distract the user. Attaching the screenshot to show the same.
Screenshot from 2021-04-11 23-00-00
As you can see naturally the eyes are getting distracted by the very prominent buttons.

3. Need to tweak the night mode as well. In the current night mode with this changes, the buttons are very hard to spot.
Screenshot from 2021-04-11 23-00-29

4. I really liked the new button colors and evenly spaced buttons and not getting pushed to left side. The animation on hower is awesome. The editpad is currently themed to look very plain, basic and give a very old age feel to user. Where application is taking a backseat and letting the user to use the editpad for taking notes. With this aesthetic of the editpad, I think we can remove some elements, and make it look something like below.
Screenshot from 2021-04-11 23-04-25
Notice the buttons are now much smaller and box shadow has been removed. One thing will be great if the buttons start from left side. There is a lot of space on left hand size on the +new button

Feel free to comment if you doesn't agree with some points I mentioned or have some different idea.

**Comment by shweshi** [comment]
@Ashish9836 Just check out the changes, looking really nice. Let me know once the PR is ready.

**Comment by ghost** [comment]
i have done all changes from my side
feel free to give any suggestion...
and you can accept the pr from my side

**Comment by shweshi** [comment]
@Ashish9836 Merged. Thanks.

**Pull request merged**